src/test/java/jeryl/fyp/logic/parser/FypManagerParserTest.java

```diff
@@ -120,6 +122,20 @@ public void parseCommand_mark() throws Exception {
         assertEquals(new MarkCommand(studentId, projectStatus), command);
     }
 
+    @Test
+    public void parseCommand_sortProjectStatus() throws Exception {
+        assertTrue(parser.parseCommand(SortProjectStatusCommand.COMMAND_WORD) instanceof SortProjectStatusCommand);
+        assertTrue(parser.parseCommand(SortProjectStatusCommand.COMMAND_WORD + " 3")
+                instanceof SortProjectStatusCommand);
+    }
+
+    @Test
+    public void parseCommand_sortSpecialisation() throws Exception {
+        assertTrue(parser.parseCommand(SortSpecialisationCommand.COMMAND_WORD) instanceof SortSpecialisationCommand);
+        assertTrue(parser.parseCommand(SortSpecialisationCommand.COMMAND_WORD + " 3")
+                instanceof SortSpecialisationCommand);
+    }
+
```

Comment:
There can be more tests here but I suppose we'll hold this until v1.4.

Reply:
Sure I'll raise an issue separately!